internal/app/web/templates/reviews/show/_triggers.html

```diff
@@ -0,0 +1,23 @@
+<section id="triggers" hx-target="this" hx-swap="outerHTML">
```

Comment:
set the `hx-target` so htmx knows that this is the place to replace when we're returning a partial from the server when adding a new bound trigger